runtimes/js/encore.dev/storage/sqldb/database.ts

```diff
@@ -329,15 +331,13 @@ export class Transaction extends BaseQueryExecutor {
   async rollback() {
     const source = getCurrentRequest();
     await this.impl.rollback(source);
+    this.done = true;
```

Comment:
I wonder if it's more correct to set this before starting the rollback than when it completes?